powershell-adapter/Tests/powershellgroup.resource.tests.ps1

```diff
@@ -28,6 +28,7 @@ Describe 'PowerShell adapter resource tests' {
         $LASTEXITCODE | Should -Be 0
         $resources = $r | ConvertFrom-Json
         ($resources | ? { $_.Type -eq 'TestClassResource/TestClassResource' }).Count | Should -Be 1
+        ($resources | Where-Object -Property type -EQ 'TestClassResource/TestClassResource').capabilities | Should -BeIn @('get', 'set', 'test', 'export')
```

Comment:
This test should be more specific.  Looking at the psd1, it should only be exactly `get` and `test`.  Maybe it makes sense to have another test resource that has `set` and `export`